recipes/jinja_partials/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "jinja_partials" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jinja_partials-{{ version }}.tar.gz
+  sha256: 5b32cc035ee93f372ddfcecf96c4837450d8561d0e2ff26c89dc0d0bd071128b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
```

Comment:
Looks like they will [bump to Python 3.8 in the next release]( https://github.com/mikeckennedy/jinja_partials/commit/285d147ad3867c4884de414fea36b928e68364d0#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R6 ). So please keep an eye on that